File: test/smokes/metrics_codeclone/option_no_skip_blocks/src/fizzbuzz.c
#include <stdio.h>

int main(void)
{
  int i;

  for (i = 1; i <= 100; i++) {
    if (i % 3 == 0 && i % 5 == 0) {
      printf("Fizz, Buzz\n");
    } else if (i % 3 == 0) {
      printf("Fizz\n");
    } else if (i % 5 == 0) {
      printf("Buzz\n");
    } else {
      printf("%d\n", i);
    }
  }

#if 0
  for (i = 1; i <= 100; i++) {
    if (i % 3 == 0 && i % 5 == 0) {
      printf("Fizz, Buzz\n");
    } else if (i % 3 == 0) {
      printf("Fizz\n");
    } else if (i % 5 == 0) {
      printf("Buzz\n");
    } else {
      printf("%d\n", i);
    }
  }
#endif

  return 0;
}

File: test/smokes/metrics_complexity/with_ignore/src/baz/fizzbuzz.py
#
# FizzBuzz in Python
#

def do_nothing(dummy):
	pass

def do_fizz_Buzz(n, dummy1, dummy2):
# FizzBuzz in Python
  for i in range(n):
    divBy3 = i % 3 == 0
    divBy5 = i % 5 == 0

# print FizzBuzz
    if divBy3 and divBy5:
        print("FizzBuzz")
    elif divBy3:
        print("Fizz")
    elif divBy5:
        print("Buzz")
    else:
        print(i)

  do_nothing(dummy1)
  print("End")

if __name__ == '__main__':
  do_fizz_Buzz(100, None, None)
